apps/site/assets/css/_events.scss

```diff
@@ -18,6 +18,100 @@
   display: inline-block;
 }
 
+.event-listing {
```

Comment:
The container that holds all of the Events Hub "guts" -- the left nav panel and the events list -- should have a reduced height, so that the "Events" title, the year dropdown, the calendar / list view toggle, and the MBTA website header are all available even after scrolling.  ZH mock:  https://zeroheight.com/2fedee66c/p/10db2c-events-hub/i/29298936

Reply:
And re: this line, possibly rename "m-event-listing"

Reply:
Thanks for flagging -- it's true my implementation doesn't quite match the mockup! [I raised it with Yinka in this ticket](https://app.asana.com/0/555089885850811/1199915858612059) and he gave me the approval for this version ;)